src/closure-externs.js

```diff
@@ -168,3 +168,28 @@ navigator.mozGamepads = function() {};
  * @return {Array.<Gamepad>}
  */
 navigator.gamepads = function() {};
+
+/**
+ * Backported from latest closure...
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/Document/currentScript
+ */
+Document.prototype.currentScript;
+
+//Atomics library (not yet in latest closure):
+//See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Atomics
+var Atomics;
+Atomics.prototype.NOTEQUAL = -1;
```

Comment:
Actually, these numbers are old specification. In the new Atomics spec, these members don't exist, but strings are used instead. See this commit: https://github.com/kripken/emscripten/commit/2762ea6a70ff1eff96434ce927c6821d3b0738e5